gitflow_toolbox/ensure_mr.py

```diff
@@ -5,31 +5,73 @@
 
 
 @click.command()
-@click.option("--remote/--current", default=False)
 @click.argument("source_branch", type=str)
 @click.argument("target_branch", type=str)
 @click.argument("title", type=str)
 @click.argument("description", type=str)
 @click.argument("labels", type=str, nargs=-1)
+@click.option(
+    "--keep-source-branch",
+    "-ksb",
```

Comment:
Use a single letter as [POSIX standard](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap12.html)
```suggestion
    "-k",
```